aioxmpp/cache.py

```diff
@@ -33,6 +33,57 @@
 
 import collections.abc
 
+class Link:
```

Comment:
FWIW (this doesn’t need changing), I don’t like the nomenclature. In my textbooks, the parts of a linked list have always been called "Node" or "Element". I find Link confusing, because I would expect it to represent the relationship between two elements, not the element itself.

Reply:
Hmm, probably you are right, node would have been a better name. My association when naming this was a link of a chain.